ow_plexil/src/plexil-adapter/OwInterface.h

```diff
@@ -127,18 +134,21 @@ class OwInterface
   void unstowAction (int id);
   void stowAction (int id);
   void grindAction (double x, double y, double depth, double length,
-               bool parallel, double ground_pos, int id);
+                    bool parallel, double ground_pos, int id);
```

Comment:
I just committed a fix that removes all the tab characters.

Reply:
Thanks! I don't know how tab characters keep on sneaking in. I think my vimrc must be screwed up.